bin/sphinx.py

```diff
@@ -11,50 +11,23 @@
         help=("If json files and time profile txt files not in the same directory, "
             "specify the base path in which the model json and time profile txt files "
             "might be found. Optional. "))
-parser.add_argument("--resume",
-        help=("Specify resume to add forecasts to an existing "
-            "dataframe and recalculate metrics."), action="store_true")
-parser.add_argument("--DataFrame", type=str, default="",
-        help=("Filename of existing Pandas DataFrame containing the results of a previous SPHINX run in pkl format. "
-            "Must specify if choose resume."))
+parser.add_argument("--resume", type=str, default="",
```

Comment:
None is a better default for an unset command line argument, or an unset keyword argument in general.